Assets/Scripts/Interaction/Storage.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using Interaction.Core;
+using UnityEngine;
+
+namespace Interaction
+{
+    [RequireComponent(typeof(InteractionReceiver))]
+    [RequireComponent(typeof(Container))]
+    public class Storage : MonoBehaviour, IInteraction
+    {
+        public bool onlyIfOpen;
+
+        private Container container;
+        private Openable openable;
+        
+        private void Start()
+        {
+            container = GetComponent<Container>();
+            openable = GetComponent<Openable>();
+        }
+
+        public void Setup(Action<string> listen, Action<string> blocks)
+        {
+            listen("store");
+            blocks("open");
```

Comment:
It does not seem like it is possible to block multiple types of events. Won't we need the ability to do that once we define enough interactions?

Reply:
Yes, you can block as many events as you want